audio-worklet/basic/js/porter.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * A simple MessagePort tester.
+ * 
+ * @class PorterProcessor
```

Comment:
Porter?  That doesn't seem to indicate very well that this is testing a MessagePort.

Reply:
Done.